jujugui/static/gui/src/app/components/user-profile/model-list/model-list.js

```diff
@@ -282,6 +282,16 @@ YUI.add('user-profile-model-list', function() {
       if (model.owner) {
         owner = model.owner.split('@')[0];
       }
+
+      const modelUser = model.users ? model.users.filter(user => {
+        return user.displayName === props.userInfo.profile;
+      }) : null;
+
+      let permission = 'unknown';
```

Comment:
How would we not know what the permissions are? Can you add a comment to this effect?